catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/IngestCommandTest.java

```diff
@@ -186,9 +171,8 @@ public void testExpectedCountsWithIgnore() throws Exception {
      * @throws Exception
      */
     @Test
-    @Ignore
-    //TODO: Use a conditional ignore from JUnit to skip this test on Windows
     public void testIgnoreHiddenFiles() throws Exception {
+        assumeFalse(SystemUtils.IS_OS_WINDOWS);
```

Comment:
Given there is no Java API to create hidden files, or to set the hidden property for a file on any platform, I would say this is fine. One note: I would make sure that whichever runner is being used, that is `@RunWith(...)`, will honor the `org.junit.Assume` API (http://junit.sourceforge.net/javadoc/org/junit/Assume.html). Might need to check the base class for that annotation. 

See also: 
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SystemUtils.html#IS_OS_WINDOWS

http://site.mockito.org/mockito/docs/current/org/mockito/runners/MockitoJUnitRunner.html

Seems like everything checks out 👍